usr/share/rear/restore/BORG/default/300_load_archives.sh

```diff
@@ -17,17 +17,47 @@ fi
 # Display header.
 LogUserOutput "
 === Borg archives list ===
-Location:   ${BORGBACKUP_HOST:-USB}
-Repository: $BORGBACKUP_REPO
-"
+
+Location:           ${BORGBACKUP_HOST:-USB}
+Repository:         $BORGBACKUP_REPO
+Number of archives: $archive_cache_lines"
 
 # Display BORGBACKUP_ARCHIVE_CACHE file content
 # and prompt user for archive to restore.
 # Always ask which archive to restore (even if there is only one).
 # This gives possibility to abort restore if repository doesn't contain
 # desired archive, hence saves some time.
+
+# Pagination for selecting archives:
+# Show BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER archives at a time, starting
+# with the current ones.
+# If no valid choice is given, cycle through older archives.
+# Enabled by default (BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER=10).
+# To disable pagination set BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER=0.
+
+archive_cache_last_shown=0
```

Comment:
If `archive_cache_last_shown` is only used in this script
it would be more clear to declare its local usage explicitly
as `local archive_cache_last_shown=0`

Reply:
Both `archive_cache_lines` and `archive_cache_last_shown` are local to this script.

See b4ac81f9 for it.